### ISSUE TITLE
CycleGAN Testing one of your pre-trained models: Is it possible to get intermediate results?

### ISSUE THREAD

**Issue by Jars-of-jam-Scheduler** [junyanz/pytorch-CycleGAN-and-pix2pix#1022]
I'm testing one of your CycleGAN pre-trained models. I've followed your instructions at https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix and it works perfectly.

However, I would want to know if it was possible to get not only the final result in the default directory ./results , but also the intermediate resulting images (the original image,  the result image, and some or all of the images between them)?

Thank you in advance!
Best regards,

**Comment by junyanz** [comment]
Which kinds of intermediate results are you referring to? The network only takes an image as input and produces the final result. There are no intermediate image results predicted by the network.
